Do not notify in case of changes due to a change in filtering (#94)
ambanum/CGUs: CONTRIBUTING.md

```
npm run validate:schema $service_id
```

> If your change filters or selectors and want to refilter documents from snapshots and regenerate versions only (`$service_id` is optional):

Reviewer: ```suggestion
If you change filters or selectors and want to refilter documents from snapshots and regenerate versions only:
```

```

> npm run refilter $service_id

Reviewer: ```suggestion
npm run refilter [$service_id]
```
